graphstorm-processing/tests/test_converter.py

```diff
@@ -56,7 +56,7 @@ def test_try_read_unsupported_feature(converter: GConstructConfigConverter, node
     node_dict["nodes"][0]["features"] = [
         {
             "feature_col": ["paper_title"],
-            "transform": {"name": "tokenize_hf"},
+            "transform": {"name": "bert_hf"},
```

Comment:
This change is aimed to trigger an expected error that we don't support this transformation yet?

Reply:
Yes. As currently we have already accepte tokenize_hf, we have to change it to bert_hf